chain/consensus/filcns/upgrades.go

```diff
@@ -1255,7 +1264,12 @@ func upgradeActorsV7Common(
 	root cid.Cid, epoch abi.ChainEpoch, ts *types.TipSet,
 	config nv15.Config,
 ) (cid.Cid, error) {
-	buf := blockstore.NewTieredBstore(sm.ChainStore().StateBlockstore(), blockstore.NewMemorySync())
+	writeStore, err := autobatch.NewBlockstore(sm.ChainStore().StateBlockstore(), blockstore.NewMemorySync(), 100_000, 100, true)
```

Comment:
100_000 here wasn't chosen very scientifically...

Reply:
This blockstore:

1. Doesn't guarantee a flush order.
2. Doesn't actually wait in `Flush` for the all data to be written.
3. Definitely wasn't written for this use-case.

3 isn't really an issue, but 1 and 2 are. Ideally the node won't be borked if you crash during a pre-migration.